gdk4/Gir.toml

```
@@ -240,15 +245,38 @@ status = "generate"
 name = "Gdk.ContentFormatsBuilder"
 status = "generate"
     [[object.function]]
-    name = "add_gtype"
-    rename = "add_type"
+    name = "free_to_formats"
+    ignore = true # marked skip, redundant with to_formats
+    [[object.function]]
+    pattern = "to_formats|add_.+"
+    manual = true # renamed, returns non-null, builder-like
 
 [[object]]
 name = "Gdk.ContentProvider"
 status = "generate"
+manual_traits = ["ContentProviderExtManual"]
     [[object.function]]
     name = "new_typed"
     ignore = true # new_for_value is what the bindings are supposed to use
+    [[object.function]]
+    name = "get_value"
+    manual = true # to return Value
```

```suggestion
    manual = true # to return Value
    doc_trait_name = "ContentProviderExtManual"
```
otherwise the manual function won't get documented

---

ok